yt/yt/server/scheduler/helpers.cpp

```diff
@@ -339,6 +339,11 @@ const std::vector<TStatisticsDescription>& GetOperationStatisticsDescriptions()
         {"user_job/gpu/max_memory_used", "Maximum registered GPU memory usage", "bytes"},
         {"user_job/gpu/memory_total", "Total available GPU memory", "bytes"},
 
+        {"job/cpu/cumulative_reserve", "Amount of CPU reserved for job execution multiplied by time in milliseconds", "core*ms"},
```

Comment:
`core*ms` does not exist as far as our UI is concerned. Let's call it CPU-time and use "ms" as the unit.